test/JDBC/input/BABEL-4424.sql

```diff
@@ -0,0 +1,104 @@
+create table babel_4359_t1 (a numeric(6,4), b numeric(6,3), c numeric);
+go
+
+insert into babel_4359_t1 values (4, 16, 1111);
```

Comment:
let us add some test cases with boundary values for numeric, also have numeric type with min/max precision/scale (from SQL server perspective)

Reply:
Added